Food/jcli.lua

```diff
@@ -0,0 +1,51 @@
+local name = "jenkins-cli"
+local version = "0.0.16"
+
+food = {
+    name = name,
```

Comment:
the name should be `jcli`, as the filename (jcli.lua) and other package managers refer to it as `jcli`.